docs/install/installing-istio.md

```diff
@@ -0,0 +1,281 @@
+    `# Disable mixer policy check, since in our template we set no policy.` \
+    --set global.disablePolicyChecks=true \
+    `# Set gateway pods to 1 to sidestep eventual consistency / readiness problems.` \
+    --set gateways.istio-ingressgateway.autoscaleMin=1 \
+    --set gateways.istio-ingressgateway.autoscaleMax=1 \
+    --set gateways.istio-ingressgateway.resources.requests.cpu=500m \
+    --set gateways.istio-ingressgateway.resources.requests.memory=256Mi \
+    `# More pilot replicas for better scale` \
+    --set pilot.autoscaleMin=2 \
+    `# Set pilot trace sampling to 100%` \
+    --set pilot.traceSampling=100 \
+    install/kubernetes/helm/istio \
+    `# Removing trailing whitespaces to make automation happy` \
+    | sed 's/[ \t]*$//' \
+    > ./istio.yaml
+
+   kubectl apply -f istio.yaml
+   ```
+
+### Installing Istio with no sidecar injection
+
+If you don't need Istio service mesh, or want to enable the service by 
+[manually injecting the Istio sidecars](https://istio.io/docs/setup/kubernetes/additional-setup/sidecar-injection/#manual-sidecar-injection), you can install an Istio without sidecar injector.
+
+1. Enter the following commands to download Istio:
+   ```shell
+   # Download and unpack Istio
+   export ISTIO_VERSION=1.1.3
+   curl -L https://git.io/getLatestIstio | sh -
+   cd istio-${ISTIO_VERSION}
+   ```
+
+1. Enter the following command to install the Istio CRDs first:
+   ```shell
+   for i in install/kubernetes/helm/istio-init/files/crd*yaml; do kubectl apply -f $i; done
+   ```
+   Wait a few seconds for the CRDs to be committed in the Kubernetes API-server,
+   then continue with these instructions.
+
+1. Enter the following command to install Istio:
+   ```shell
+   # A lighter template, with no sidecar injection.
+   helm template --namespace=istio-system \
+    --set sidecarInjectorWebhook.enabled=false \
+    --set global.proxy.autoInject=disabled \
+    --set global.omitSidecarInjectorConfigMap=true \
+    --set global.disablePolicyChecks=true \
+    --set prometheus.enabled=false \
+    `# Disable mixer prometheus adapter to remove istio default metrics.` \
+    --set mixer.adapters.prometheus.enabled=false \
+    `# Disable mixer policy check, since in our template we set no policy.` \
+    --set global.disablePolicyChecks=true \
+    `# Set gateway pods to 1 to sidestep eventual consistency / readiness problems.` \
+    --set gateways.istio-ingressgateway.autoscaleMin=1 \
+    --set gateways.istio-ingressgateway.autoscaleMax=1 \
+    `# Set pilot trace sampling to 100%` \
+    --set pilot.traceSampling=100 \
+    install/kubernetes/helm/istio \
+    `# Removing trailing whitespaces to make automation happy` \
+    | sed 's/[ \t]*$//' \
+    > ./istio-lean.yaml
+
+  kubectl apply -f istio-lean.yaml
+  ```
+
+### Installing Istio with Secret Discovery Service
+
+[Secret Discovery Service](https://istio.io/docs/tasks/traffic-management/secure-ingress/sds/)
```

Comment:
@ZhiminXiang 
Similar point to above; can we give a use case example for why someone might want to use SDS?

Reply:
Thanks for mentioning this!
With SDS enabled in ingress gateway, users can configure Gateway TLS more flexibly.
1. Users can dynamically update the Gateway TLS with multiple TLS certificates to terminate TLS connection. See [example](https://istio.io/docs/tasks/traffic-management/secure-ingress/sds/)
2. If users want to use [Auto TLS feature] of Knative, they need to install Istio with SDS in the gateway. See [instructions of auto TLS](https://github.com/knative/docs/pull/1281/files#diff-1ac0a9c578d517c34bf1c688503c8733R14)